### DISCUSSION TITLE
2022.08.02 MSF Delegates Meeting 19:30 CET

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/MSF-Delegates#25] [General]
We'll be meeting again in the discord weekly voice channel and the plan is to pick up with last week's discussions (meeting notes: omigroup/omigroup#256).

Tentative agenda:

- Administrative follow-ups (10 minutes)
  - Review meeting time, cadence, availabilities, OMI subgroup status.
  - Per-user instructions for creating MSF admin+nickname accounts.
  - Forum updates:
    - First "Standards Registry" meeting.
    - MSF released a membership banner pack.
    - Note upcoming forum meetings:
      - Plenary Meeting # 3 (August 3rd 7AM Pacific)
      - Standards Registry Exploratory Group # 2 (August 17th 7AM Pacific)
- Gather agenda points for next meeting (5 minutes)
- Miro jam session (45 minutes)
  - general brainstorming (eg: diagram ecosystem/participation/community strategies)

**Comment by funksoup** [comment]
Next MSF Plenary: Wed, 3 August at 7:00AM PT / 14:00 UTC (@humbletim offered to  stream it again, probably on [#hack-chat](https://discord.com/channels/770382203782692945/846141671712358420))

Proposed new OMI-MSF delegates' meeting time: Wednesdays at 8:00am PT / 15:00 UTC

OMI Town Hall: Wednesdays at 9:00AM PT / 16:00 UTC

OMI subgroup status: yes, create OMI subgroup for MSF-delegates, include 'scope of work'/statement draft + proposed time as proposal

Next MSF meeting in which we'll whiteboard/brainstorm on Miro: Wed, 17 August, 8:00am PT / 15:00 UTC
(perhaps on OMI's Miro board?)

**Reply by pcsmgnt** [reply]
When does the new schedule for the MSF-delegates on Wednesdays start? I note this weeks would clash with the proposed Plenary Meeting # 3. I guess next week?

**Reply by mikeskydev** [reply]
The proposed time would be directly after any plenary meetings or working groups provided by the MSF, hopefully this would be the perfect time for a discussion about it!

**Comment by funksoup** [comment]
**Draft of MSF delegate participation / scope of work:**

As members the Open Metaverse Interoperability Group (OMI) and Metaverse Standards Forum delegates, we will represent the interests of the OMI community by

1. upholding the OMI’s group values, as stated on omigroup.org
2. communicating any standards creation, policymaking and other relevant actions of the Metaverse Standards Forum (MSF)
3. maintaining knowledge of OMI community needs by attending OMI meetings regularly [define 'regularly' at some point?]
4. maintaining knowledge of MSF activity by attending MSF meetings regularly
5. documenting MSF-related activity at [ github repo?] for the benefit of the OMI community
6. inquiring about OMI community needs through outreach (i.e., polls, discussions, etc.) as needed
7. fostering standards creation, policymaking and other relevant actions that support OMI’s group values as well as the overall values consistent with open metaverse goals
8. challenging standards creation, policymaking and other relevant actions that do not support OMI’s group values nor the overall values consistent with open metaverse goals

**Comment by pcsmgnt** [comment]
Do we have a dedicated repo? I know this has come up before, but I am starting to think it might be worth having one so we can reference it in the inks channel for anyone interested in updates. it will also allow Jesse to configure the webhook
